test/cpp/jit/test_backend_lib.cpp

```diff
@@ -6,12 +6,17 @@ namespace jit {
 // necessary to test that the JIT backend registration endpoints and
 // code generation are working correctly. It is not intended to
 // produce numerically correct results.
+template <bool isAvailable>
```

Comment:
What's the purpose of this template argument? To decide if this backend be built or not at compilation time?